src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -306,7 +306,7 @@ fn fat_lto(
         drop(linker);
         save_temp_bitcode(&cgcx, &module, "lto.input");
 
-        // Internalize everything that *isn't* in our whitelist to help strip out
+        // Internalize everything that *isn't* in our allowlist to help strip out
         // more modules and such
         unsafe {
             let ptr = symbol_white_list.as_ptr();
```

Comment:
Maybe name this `exported_symbol_names` (this variable occurs in several different functions in this file)